clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -21,6 +21,20 @@
    :type       type-name
    :attributes (dissoc value :id)})
 
+(defn- update-vals [map vals f]
+  (reduce #(update-in %1 [%2] f) map vals))
+
+(defn- parse-date [value]
```

Comment:
It's pretty funny to think that we're thinking that a 70 line file is "growing a bit large".  :)  I do think that there's probably room to start moving some of the code into a 'util' namespace.  I'll see how things look when I go to implement the next case.